base/initdefs.jl

```diff
@@ -21,17 +21,18 @@ const ARGS = String[]
 """
     exit([code])
```

Comment:
I'd drop the square brackets if you're going to use julia syntax for default arguments (which I think is clearer anyway)